src/cosmosdb-preview/azext_cosmosdb_preview/custom.py

```diff
@@ -566,7 +567,11 @@ def cli_cosmosdb_managed_cassandra_datacenter_create(client,
         base64_encoded_cassandra_yaml_fragment=base64_encoded_cassandra_yaml_fragment
     )
 
-    return client.begin_create_update(resource_group_name, cluster_name, data_center_name, data_center_properties)
+    data_center_resource = DataCenterResource(
+        data_center_properties=data_center_properties
```

Comment:
Is this a bug in sdk generation or was something missed in Swagger ?? Did you make changes to the sdk manually ?? Ideally we don't want to do that, we should always fix in swagger and generate sdk from swagger.

Reply:
We haven't changed the Swagger, so it has to be a bug in SDK generation.